setup.py

```diff
@@ -47,11 +42,19 @@
     include_package_data=True,
 
     # Allows `setup.py test` to work correctly with pytest
-    setup_requires=['numpy>=1.19.5'] + pytest_runner,
+    setup_requires=[] + pytest_runner,
 
     # Additional entries you may want simply uncomment the lines you want and fill in the data
     # url='http://www.my_package.com',  # Website
-    # install_requires=[],              # Required packages, pulls from pip if needed; do not use for Conda deployment
+    install_requires = [
+        'numpy>=1.16.0',
+        'mdanalysis>=2.0.0b0',
+    ],
+
+    # packages required to run tests
+    tests_require = [
```

Comment:
You can add a `tests_require` entry to force it to make sure that `pytest` is available if you want to run the tests (not super necessary, but it's nice enough to have).